newton/tests/test_viewer_rerun_init_args.py

```
@@ -0,0 +1,104 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import warnings
+from unittest.mock import Mock, patch
+
+# ruff: noqa: PLC0415
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)